requirements.txt

```diff
@@ -1,3 +1,11 @@
 Django==1.7.1
-wsgiref==0.1.2
 Pillow==2.7.0
+PyJWT==0.4.1
+argparse==1.2.1
```

Comment:
Actually these were dependencies that python-social-auth app needs and installed.